public/assets/scripts/fetch-function-doc.mjs

```diff
@@ -104,6 +104,19 @@ function loadDocumentContent(event) {
 }
 
 function loadFunctionDoc(doc) {
+    const functionName = doc.data().functionName;
+    const headBanner = document.querySelector("head-banner");
+
+    // Hide the main title, tagline, and button within the head-banner
+    const titleElement = headBanner.querySelector(".page-name");
+    const tagLineElement = headBanner.querySelector("h2");
+    const buttonElement = headBanner.querySelector("button");
+
+    if (titleElement) titleElement.style.display = "none";
+    if (tagLineElement) tagLineElement.style.display = "none";
+    if (buttonElement) buttonElement.style.display = "none";
```

Comment:
The exact same code exists in `loadDomainPage`, so best to put this into a helper function.